src/actions/guides/testing/ci-cd.cr

```diff
@@ -0,0 +1,60 @@
+class Guides::Testing::Introduction < GuideAction
+    guide_route "/testing/ci-cd"
+  
+    def self.title
+      "Setting up Continuous Integration"
+    end
+  
+    def markdown : String
+        <<-MD
+        ## Setting up Continuous Integration
+    
+        Continuous integration (CI) is a system that automatically
+        runs your tests every time you check in new code. That way,
+        you can pinpoint the exact change you made that broke the
+        build.
+
+        There are lots of platforms that you can set up to 
+        automatically run tests for you. Below is a list of systems 
+        that are free and popular for open source projects:
+
+        - [Travis CI](https://travis-ci.org/) - The `lucky` CLI 
+            already generates a config file to set up Travis to run 
+            your tests for you
+        - [Circle CI](https://circleci.com/)
+        - [GitHub actions](https://github.com/features/actions)
+
+        
+        There are pros and cons to each system and your choice depends
+        on a variety of factors. We won't be describing how to choose
+        here, we'll be explaining how to set each of these up if you
+        choose to.
+
+        ## Travis CI
+
+        Lucky already generates a config file for 
+        [Travis CI](https://travis-ci.org/), so if you choose to use
+        that platform, congratulatioins! You're done.
```

Comment:
How about something like:

> Lucky generates the TravisCI config file for you when you create your app.

But you're not really done at that point, right? Or are you? I don't really know how that works. Does github just sync those up when you have that config file?